packages/actor-scraper/playwright-scraper/INPUT_SCHEMA.json

```diff
@@ -118,6 +118,12 @@
             "groupCaption": "Browser masking",
             "groupDescription": "Settings that help mask as a real user and prevent scraper detection."
         },
+        "headful": {
+            "title": "Run browsers in headful mode",
+            "type": "boolean",
+            "description": "By default, browsers run in headless mode. You can toggle this on to run them in headful mode.",
```

Comment:
I would just add a sentence as `Headful mode can help with certain rare anti-scraping protections but is slower and more costly.`